pkg/systemd/page-status.jsx

```diff
@@ -55,7 +55,7 @@ function get_pficon(name) {
     if (name == "check")
         return <CheckIcon color="green" data-pficon={name} />;
     if (name == "spinner")
-        return <Spinner isSVG size="sm" data-pficon={name} />;
+        return <Spinner isSVG size="md" data-pficon={name} />;
```

Comment:
This is kind of unnecessary, as we redefine the size in the CSS, but md is closer than sm.